ajax/libs/TypewriterJS/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "TypewriterJS",
+  "version": "1.0.0",
+  "filename": "typewriter.min.js",
+  "description": "A simple yet powerful native javascript plugin for a cool typewriter effect.",
```

Comment:
@sufuf3 You're right about the `description`. However, the homepage comment in the JS file links to to the authors website and not specifically to the library. https://safi.me.uk/typewriterjs/ links to the library page so I think that's better to use?